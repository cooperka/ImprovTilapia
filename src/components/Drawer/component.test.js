import React from 'react';
import { shallow } from 'enzyme';

import Component from './component';

jest.mock('ScrollView', () => require.requireMock('ScrollViewMock'));
jest.mock('./DrawerItems/component', () => 'DrawerItems');

it('renders as expected', async () => {
  const wrapper = shallow(<Component />);
  expect(wrapper).toMatchSnapshot();
});
