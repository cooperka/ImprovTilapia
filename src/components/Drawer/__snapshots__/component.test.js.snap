// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<ScrollView>
  <SafeAreaView
    forceInset={
      Object {
        "horizontal": "never",
        "top": "always",
      }
    }
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <DrawerItems />
  </SafeAreaView>
</ScrollView>
`;
